bindings/psat.h

```diff
@@ -174,28 +162,21 @@ NAN_METHOD(resultsExisting) {
 	try {
         auto const & ex = psat.calculateExisting();
 
-        std::unordered_map<std::string, double> out = {
-                {"pump_efficiency",          ex.pumpEfficiency * 100},
-                {"motor_rated_power",        ex.motorRatedPower},
-                {"motor_shaft_power",        ex.motorShaftPower},
-                {"pump_shaft_power",         ex.pumpShaftPower},
-                {"motor_efficiency",         ex.motorEfficiency * 100},
-                {"motor_power_factor",       ex.motorPowerFactor * 100},
-                {"motor_current",            ex.motorCurrent},
-                {"motor_power",              ex.motorPower},
-                {"load_factor",              ex.loadFactor},
-                {"drive_efficiency",         ex.driveEfficiency * 100},
-                {"annual_energy",            ex.annualEnergy},
-                {"annual_cost",              ex.annualCost * 1000},
-                {"annual_savings_potential", psat.getAnnualSavingsPotential() * 1000},
-                {"optimization_rating",      psat.getOptimizationRating()}
-        };
-
-        for (auto const &p: out) {
-            Local <String> key = Nan::New<String>(p.first).ToLocalChecked();
-            Local <Number> value = Nan::New(p.second);
-            Nan::Set(r, key, value);
-        }
+        SetR("pump_efficiency",ex.pumpEfficiency * 100);
```

Comment:
Need to remove arithmetic from the binding functions. `ex.pumpEfficiency * 100` should just be `ex.pumpEfficiency`. However, we need to reflect this calculation in the calculate function. In this case, the calculation happens in `psat.calculateExisting();`.